src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -365,17 +374,64 @@ private static Request parseNativeRequest(String rawStringNativeRequest, int imp
         }
     }
 
-    private void validateNativeContext(Integer context, int index) throws ValidationException {
-        if (context != null && (context < 1 || context > 3)) {
+    private void validateNativeContextTypes(Integer context, Integer contextSubType, int index)
+            throws ValidationException {
+        if (context != null && (context < ContextType.CONTENT.getValue() || context > ContextType.PRODUCT.getValue())) {
```

Comment:
Implicit unboxing here and in the code that follows is a bit of concern to me. Unnecessary unboxing could be avoided if `context` and `contextSubType` values were saved in `int` local variables first and those local variables were used in following checks. It seems that `0` is safe to use if corresponding `Integer` is `null`.